### PR TITLE
OMG-194 Generate swagger spec

### PR DESCRIPTION
**Pull request by pgebal**

### CODE REVIEW

File: apps/omg_watcher/lib/web/controllers/utxo.ex [hunk 1]
```diff
@@ -70,4 +71,96 @@ defmodule OMG.Watcher.Web.Controller.Utxo do
...
+
+    parameters do
+      address(:query, :string, "Address of utxo owner", required: true)
+    end
+
+    response(200, "OK", Schema.ref(:Utxos))
+  end
+
+  swagger_path :compose_utxo_exit do
+    get("/account/utxo/compose_exit")
```
Comment (pgebal) [inline apps/omg_watcher/lib/web/controllers/utxo.ex]: This resource should be named differently. Maybe account/exits/:utxopos would work better? We would treat utxopos like an exit id.

File: apps/omg_watcher/test/eventer/core_test.exs [hunk 2]
```diff
@@ -28,11 +28,10 @@ defmodule OMG.Watcher.Eventer.CoreTest do
   @tag fixtures: [:alice, :bob]
   test "notify function generates 2 proper address_received events", %{alice: alice, bob: bob} do
     recovered_tx =
-      API.TestHelper.create_recovered(
-        [{1, 0, 0, alice}, {2, 0, 0, bob}],
-        API.Crypto.zero_address(),
-        [{alice, 100}, {bob, 5}]
-      )
+      API.TestHelper.create_recovered([{1, 0, 0, alice}, {2, 0, 0, bob}], API.Crypto.zero_address(), [
```
Comment (pnowosie) [inline apps/omg_watcher/test/eventer/core_test.exs]: This doesn't look like `formatter` change - make sure you run formatter on this file

File: apps/omg_watcher/lib/web/controllers/transaction.ex [hunk 3]
```diff
@@ -52,4 +53,68 @@ defmodule OMG.Watcher.Web.Controller.Transaction do
...
+
+  def swagger_definitions do
+    %{
+      Transaction:
+        swagger_schema do
+          title("Transaction")
+
+          properties do
+            txid(:string, "Transaction id", required: true)
+            blknum1(:integer, "Childchain block number of first input utxo", required: true)
```
Comment (pnowosie) [inline apps/omg_watcher/lib/web/controllers/transaction.ex]: Pls add `the` before each `first`, `second`, e.g. "Childchain block number of the first input utxo"

Comment (pgebal) [reply apps/omg_watcher/lib/web/controllers/transaction.ex]: done

File: apps/omg_watcher/test/web/controllers/utxo_test.exs [hunk 4]
```diff
@@ -111,9 +111,9 @@ defmodule OMG.Watcher.Web.Controller.UtxoTest do
     {:ok,
      %{
        utxo_pos: _utxo_pos,
-       txbytes: _tx_bytes,
+       txbytes: txbytes,
```
[the remark this answers is not in the document]
Comment (pgebal) [reply apps/omg_watcher/test/web/controllers/utxo_test.exs]: done.

File: apps/omg_watcher/test/eventer/core_test.exs (hunk 2 above)
Comment (pgebal) [reply apps/omg_watcher/test/eventer/core_test.exs]: It was the formatter.

**Pull request merged**